components/schema-migrator/update-expected-schema-version.sh

```diff
@@ -36,7 +36,7 @@ do
     esac
 done
 
-LATEST_SCHEMA=$(ls -lr migrations/${MIGRATION_PATH} | head -n 2 | tail -n 1 | tr -s ' ' | cut -d ' ' -f9 | cut -d '_' -f1)
+LATEST_SCHEMA=$(ls migrations/${MIGRATION_PATH} | tail -n 1 | grep -o -E '^[0-9]+' | sed -e 's/^0\+//')
```

Comment:
Could you please elaborate in what context this change was needed?

Reply:
If you trace what both command do they extract the latest schema version but the "old" one did it with a lot unnecessary "pipes" that didn't apply any changes and it was difficult to understand what every pipe do